src/test/robot/api/storage/pravega.robot

```diff
@@ -92,19 +95,33 @@ Batch Create Event Stream Test
     ${args} =  Catenate  SEPARATOR= \\\n\t
     ...  --storage-namespace=scope5
     ...  --storage-driver-event-batch
+    ...  --storage-net-node-addrs=${node_addr}
     ...  --load-op-limit-count=${count_limit}
     ...  --load-step-id=${step_id}
     ...  --load-batch-size=1234
-    ...  --item-output-path=eventsStream1
     ...  --item-data-size=123
     &{env_params} =  Create Dictionary
     ${std_out} =  Execute Mongoose Scenario  ${DATA_DIR}  ${env_params}  ${args}
     Log  ${std_out}
     Validate Metrics Total Log File  ${step_id}  CREATE  ${count_limit}  0  12300000
 
+Read Events Test
+    ${step_id} =  Set Variable  read_events_test
+    Remove Directory  ${LOG_DIR}/${step_id}  recursive=True
+    ${args} =  Catenate  SEPARATOR= \\\n\t
+    ...  --load-step-id=${step_id}
```

Comment:
add the command line option
`...  --load-op-limit-count=${count_limit}`

Reply:
done